styles/components/_lightsaber.scss

```diff
@@ -0,0 +1,36 @@
+.lightsaber {
+  align-items: center;
+  display: flex;
+  flex-basis: 100%;
+  order: -1;
```

Comment:
Please order scss files according to these:
// Positioning
// Display & Box Model
// Color
// Text
// Other